PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/subgraph/SubgraphAlgorithmManager.java

```diff
@@ -21,6 +21,8 @@
  */
 public class SubgraphAlgorithmManager {
 
+  private static int dummyRootNodeId = Integer.MAX_VALUE;
```

Comment:
We can still use `Integer.MAX_VALUE / 2 ++`, `Integer.MAX_VALUE / 2 -1 --` and the same for `MIN_VALUE` :)